### ISSUE TITLE
Accessibility: keyboard 2.1.1

### ISSUE THREAD

**Issue by chrismplus** [mplusmuseum/mplusmuseum-stories#172]
_Please cross-reference against Web Accessibility Evaluation Report, p. 7._

**Issue 001: Clickable items cannot be operated with keyboard**
- Note: @rdunk — when I try to access the website sections via keyboard I can't. Can you confirm? 

**Issue:** Screen reader user cannot close the menu with the Enter key.

Chris following up with vendor.

**Comment by chrismplus** [comment]
@rdunk 

Further clarification from HKBU: 

> It only occurs when screen reading software is on, you may refer to the video as follows,
> https://files.kapwing.com/final_5a5811c7108ad400149467e4.mp4
>  
> Adding the highlight effect to keyboard is highly appreciated.
> We can open the links in the menu without problem. Just for your information, we are using Chrome.

**Comment by rdunk** [comment]
I'm not able to see this view that link unfortunately, but I believe a highlight effect has been added to menu items. Will check back on this one.

**Comment by chrismplus** [comment]
@rdunk looks like the link's expired. I'll move this over to review because it looks like it's fixed.

**Comment by chrismplus** [comment]
Hi @rdunk — here's the video link for this one: http://demo.hkbu.org.hk/close_menu.mp4

Comment from HKBU: 
![image](https://user-images.githubusercontent.com/26731347/35610377-0524c2aa-069c-11e8-9e47-508265e5c474.png)

**Comment by rdunk** [comment]
That's strange, I'm able to close the menu using both the Enter and Esc keys whilst I have screenreader software enabled. Perhaps this is to do with the specific screen-reader software? Did they provide any more information about that?

**Comment by chrismplus** [comment]
@rdunk — checking with the HKBU re: screenreader software.

**Comment by chrismplus** [comment]
@rdunk from HKBU:

Testing took place in the following environment:

> NVDA 2017.4
> Google Chrome 64.0.3282.140 (64 bit)
> Windows 7 (64 bit)

> We can close the menu with ESC, but not with the close button Thanks so much!

**Comment by chrismplus** [comment]
@rdunk — let me know whether you can replicate.

**Comment by rdunk** [comment]
I've pushed an attempted fix but can't be sure it will resolve the issue, unfortunately I can't replicate this as we only have access to virtual browsers for Windows and can't install NVDA on macOS.

**Comment by chrismplus** [comment]
Re-checking with HKBU. Assigning to @chrismplus

**Comment by chrismplus** [comment]
It works perfectly now on Chrome with NVDA. The menu can be closed with the button with keyboard.
